### PR TITLE
Switch medum of exchange from tokens to ETH

### PR DESCRIPTION
**Pull request by haydenadams**

### CODE REVIEW

File: contracts/ETHWallet.sol [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    mapping (address => uint256) ethBalances;
+
+    function quickDeposit () public payable {
+        require(msg.value != 0);
+        ethBalances[msg.sender] = ethBalances[msg.sender].add(msg.value);
+    }
+
+
+    function depositToAddress (address _address) public payable {
```
Comment (jtremback) [inline contracts/ETHWallet.sol]: nice! this will be good for the bootstrapping process

File: contracts/PaymentChannels.sol [hunk 2]
```diff
@@ -102,13 +102,13 @@ contract PaymentChannels is ECVerify, MintableToken {
     function incrementBalance(address _addr, uint _value)
         internal
     {
-        balances[_addr] = balances[_addr].add(_value);
+        ethBalances[_addr] = ethBalances[_addr].add(_value);
```
Comment (jtremback) [inline contracts/PaymentChannels.sol]: Maybe this function should be in EthWallet

Comment (jtremback) [reply contracts/PaymentChannels.sol]: Kinda bugs me that ethBalances is just kind of floating around

Comment (haydenadams) [reply contracts/PaymentChannels.sol]: Good idea, I'll add this change in the next PR

**Pull request merged**